File: python/day5.py
# n = int(input("enter the array size: "))
# arr = []
# for i in range(n):
#     arr.append(int(input("enter the aarray elements:")))

# arr.sort()
# print("minimum number: ",arr[0])
# print("maximum number: ",arr[n-1])

# big=small=0
# for i in range(n):
#     for j in range(n):
#         c=1
#         if(arr[i] < arr[c]):
#             small=arr[i]
#         else:
#             big = arr[c]
# print("minimum number: ",small)
# print("maximum number: ",big)

# problem 2
# res =1
# n = int(input("enter the number: "))
# if 0<n:
#   for i in range(1,n+1):
#       res *= i
#   print(res)
# else:
#   print("invalid input")

# problem 3
# def fibonacci(n):
#     if n == 1 or n == 2:
#         return 1
#     else:
#         return fibonacci(n-1) + fibonacci(n-2)
    
# n = int(input("enter the number: "))
# res = fibonacci(n)
# print(res%100)

# a,b=0,1
# if n == 1:
#     b=0
# elif n == 2:
#     b=1
# else:
#     for i in range(2,n+1):
#         a,b=b,(a +b)%100

# print("output ",b)

#problem 4
# import math
# x = int(input("enter the number: "))
# r = math.sqrt(x)
# s = int(r)
# if r == s:
#     print(int(r))
# else:
#     print(s)
res=0
n = int(input("enter the number: "))
# res=n*(n-1)//2
# res = (1//(n+1))(2*n//n)

print(res)

File: c program/day5.c
#include<stdio.h>
int main(){
    //problem 1
    int n,a[20],t;
    printf("Enter the array size\n");
    scanf("%d",&n);
    printf("Enter the array elements:\n");
    for(int i=0; i<n; i++)
        scanf("%d",&a[i]);
    
    for(int i=0; i<n; i++){
        for(int j=i+1; j<n; j++){
            if(a[j] > a[j+1]){
                t = a[j];
                a[j] = a[j+1];
                a[j+1] = t;
            }
        }
    }
int count=0;
    printf("output\n");
    for(int i=0; i<n; i++){
        if(a[i] == a[i+1]){
            printf("%d ",a[i]);
            i++;
        }
        else
            printf("%d ",a[i]);
        count++;
    }
    printf("\n%d",count);
    //problem 2
    // for(int i=0; i<5; i++){
    //     for(int j=1; j<=i; j++){
    //         printf("*");
    //     }
    // printf("\n");
    // }

    //problem 3
    // for(int i=0; i<3; i++)
    //     printf("1010101\n");

    // printf("\n---------------\n");
    // for(int i=0; i<3; i++){
    //     for(int j=1; j<=3; j++){
    //         printf("10");
    //     }
    // printf("1\n");
    // }
    return 0;
}